tsconfig.base.json

```diff
@@ -1552,6 +1552,8 @@
       "@kbn/search-indices/*": ["x-pack/plugins/search_indices/*"],
       "@kbn/search-inference-endpoints": ["x-pack/plugins/search_inference_endpoints"],
       "@kbn/search-inference-endpoints/*": ["x-pack/plugins/search_inference_endpoints/*"],
+      "@kbn/search-navigation": ["x-pack/plugins/search_solution/search_navigation"],
```

Comment:
Should we name this `search` instead of `search_solution` and keep name consistency with the one for packages? `x-pack/packages/search`

Reply:
so I was using `observability_solution` & `observability_solution` as the examples for this and `search` seemed too generic.

Reply:
Thats fair. `security` is the other one. I dont mind either but would be good we keep consistency and rename the other folder in packages (whilst theres only one package)